js/data/bucket.js

```diff
@@ -42,6 +42,13 @@ Bucket.create = function(options) {
 Bucket.EXTENT = 8192;
 
 /**
+ * The maximum size of the arrays in an "array group."
+ * @private
+ * @readonly
+ */
+Bucket.MAX_ARRAY_LENGTH = 65535;
```

Comment:
readability++

I amended the definition to include some extra context about this constant:
```
/**
 * The maximum size of a vertex array. This limit is imposed by WebGL's 16 bit addressing of vertex buffers. 
 * @private
 * @readonly
 */
Bucket.MAX_VERTEX_ARRAY_LENGTH = Math.pow(2, 16) - 1;
```